contracts/feature-tests/use-module/tests/gov_module_whitebox_test.rs

```diff
@@ -0,0 +1,603 @@
+use multiversx_sc::types::{Address, ManagedVec, MultiValueEncoded};
+use multiversx_sc_modules::governance::{
+    governance_configurable::GovernanceConfigurablePropertiesModule, governance_proposal::VoteType,
+    GovernanceModule,
+};
+use multiversx_sc_scenario::{
+    managed_address, managed_biguint, managed_buffer, managed_token_id, rust_biguint,
+    scenario_model::{
+        Account, AddressValue, CheckAccount, CheckStateStep, ScCallStep, ScDeployStep, SetStateStep,
+    },
+    ScenarioWorld, WhiteboxContract,
+};
+
+const GOV_TOKEN_ID_EXPR: &str = "str:GOV-123456";
+const GOV_TOKEN_ID: &[u8] = b"GOV-123456";
+const QUORUM: u64 = 1_500;
+const MIN_BALANCE_PROPOSAL: u64 = 500;
+const VOTING_DELAY_BLOCKS: u64 = 10;
+const VOTING_PERIOD_BLOCKS: u64 = 20;
+const LOCKING_PERIOD_BLOCKS: u64 = 30;
+
+const INITIAL_GOV_TOKEN_BALANCE: u64 = 1_000;
+const GAS_LIMIT: u64 = 1_000_000;
+
+const USE_MODULE_ADDRESS_EXPR: &str = "sc:use-module";
+const USE_MODULE_PATH_EXPR: &str = "file:output/use-module.wasm";
+
+const OWNER_ADDRESS_EXPR: &str = "address:owner";
+const FIRST_USER_ADDRESS_EXPR: &str = "address:first-user";
+const SECOND_USER_ADDRESS_EXPR: &str = "address:second-user";
+const THIRD_USER_ADDRESS_EXPR: &str = "address:third-user";
+
+pub struct Payment {
+    pub token: Vec<u8>,
+    pub nonce: u64,
+    pub amount: u64,
+}
+
+fn world() -> ScenarioWorld {
+    let mut blockchain = ScenarioWorld::new();
+    blockchain.set_current_dir_from_workspace("contracts/features-tests/use-module");
+
+    blockchain.register_contract(USE_MODULE_PATH_EXPR, use_module::ContractBuilder);
+    blockchain
+}
+
+fn setup() -> ScenarioWorld {
+    let mut world = world();
+
+    world.set_state_step(
+        SetStateStep::new()
+            .put_account(
+                OWNER_ADDRESS_EXPR,
+                Account::new()
+                    .nonce(1)
+                    .esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(INITIAL_GOV_TOKEN_BALANCE)),
```

Comment:
I think it should work without the `rust_biguint!` we should have direct conversion from small int